libsuseconnect/libsuseconnect.go

```diff
@@ -234,10 +238,17 @@ func errorToJSON(err error) string {
 			s.Message = ierr.Error()
 			// ruby version doesn't have this but it might be useful
 			s.Data = certToPEM(ce.Certificate)
+		} else if _, ok := ierr.(*net.OpError); ok {
+			s.ErrType = "NetError"
+			s.Message = ierr.Error()
 		} else {
 			connect.Debug.Printf("url.Error: %T: %v", ierr, err)
 			s.Message = err.Error()
 		}
+	} else if je, ok := err.(connect.JSONError); ok {
+		uerr := errors.Unwrap(je)
```

Comment:
`uerr` was used for something else above. Maybe something like `jerr` would be clearer. Or maybe better not use a temp variable:
`s.Message = errors.Unwrap(je).Error()`

Reply:
done